packages/application/src/shell.ts

```diff
@@ -338,6 +338,31 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
       rootLayout.insertWidget(2, this._menuHandler.panel);
     }
 
+    // Skip Links
+    const skipLinkWidget = (this._skipLinkWidget = new Widget());
+    const skipToFilterFiles = document.createElement('a');
+    const toggleSkipLink = (toShow: boolean) => {
+      skipLinkWidget.node.classList.toggle('jp-skiplink--focus', toShow);
+    };
+    skipToFilterFiles.href = '#';
+    skipToFilterFiles.tabIndex = 1;
+    skipToFilterFiles.text = 'Skip to Filter Files';
+    skipToFilterFiles.className = 'skip-link';
+    skipToFilterFiles.addEventListener('focus', e => toggleSkipLink(true));
```

Comment:
It seems like you might be able to use focus-within instead of these handlers: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within

Reply:
Thanks. I'll try that!